dist/index.html

```diff
@@ -0,0 +1 @@
02 179.276C180.176 184.11 191.834 183.344 191.834 183.344V183.425C195.573 183.465 195.653 183.224 201.603 182.579C207.553 181.935 212.055 179.438 216.316 176.215C220.618 173.033 224.597 171.986 227.813 171.543C231.07 171.06 238.145 171.704 238.708 174.363C239.27 177.021 236.979 177.585 236.979 177.585C237.059 176.094 235.894 175.853 235.13 175.813C234.688 175.772 234.205 175.974 233.964 176.135C233.803 176.215 233.683 176.296 233.602 176.417C233.24 176.94 233.16 177.746 233.08 178.35C233.039 178.672 233.12 178.954 233.281 179.236C235.813 183.344 244.617 183.425 247.19 183.344L261.019 162.561L240.517 153.982L191.794 133.44L143.072 153.982L122.57 162.561L136.439 183.385C139.534 183.465 139.936 183.224 145.524 182.619Z" fill="white"/><path d="M399.186 193.656H2.81396V202.476H399.186V193.656Z" fill="white"/><path d="M32.6424 220.158L41.8884 229.463V248.554L38.592 251.857L41.8884 255.079V274.09L32.6424 283.354H2.81396V220.158H32.6424ZM14.874 245.735H29.7882V232.282H14.874V245.735ZM29.8284 257.778H14.874V271.231H29.8284V257.778Z" fill="white"/><path d="M58.4106 283.314H46.3506L63.315 220.199H75.375L91.3746 283.314H79.3548L76.6212 272.56H61.305L58.4106 283.314ZM69.1842 243.238L64.5612 260.476H73.566L69.1842 243.238Z" fill="white"/><path d="M124.499 220.158V232.242H112.841V283.274H100.781V232.282H89.1636V220.199H124.499V220.158Z" fill="white"/><path d="M166.589 220.158V232.242H154.931V283.274H142.871V232.282H131.253V220.199H166.589V220.158Z" fill="white"/><path d="M185.443 271.19H205.342V283.274H173.383V220.158H185.443V271.19Z" fill="white"/><path d="M244.094 220.158V232.242H224.155V245.694H240.115V257.778H224.155V271.231H244.094V283.314H212.095V220.158H244.094Z" fill="white"/><path d="M280.677 220.158L289.923 229.463V241.707H277.903V232.282H262.908V245.735H280.435L286.666 251.817L286.626 251.857L289.923 255.079V274.09L280.677 283.354H260.094L250.848 274.09V261.806H262.908V271.231H277.863V257.778H259.853L250.848 248.554V229.463L260.094 220.158H280.677Z" fill="white"/><path d="M322.163 220.158H334.223V283.274H322.163V257.778H308.736V283.314H296.676V220.158H308.736V245.694H322.163V220.158Z" fill="white"/><path d="M340.977 220.158H353.037V283.274H340.977V220.158Z" fill="white"/><path d="M389.618 220.158L398.864 229.463V250.044L389.618 259.308H371.85V283.314H359.79V220.158H389.618ZM386.764 232.282H371.85V247.265H386.764V232.282Z" fill="white"/></svg><nav class="menu"><ul class="menu__items"><li class="menu__item"><a href="game.html" class="menu__link --line">Start Game</a></li><li class="menu__item"><a href="howtoplay.html" class="menu__link --line">How to Play</a></li><li class="menu__item"><a href="highscores.html" class="menu__link --line">High Scores</a></li><li class="menu__item"><a href="settings.html" class="menu__link --line">Settings</a></li><li class="menu__item"><a href="authors.html" class="menu__link --line">Authors</a></li></ul></nav></main><script type="module" src="main.js"></script><script defer="defer" src="index.js"></script></body></html>
```

Comment:
Remove the whole `dist` folder and push a commit with it deleted. After that, add `dist` to `.gitignore`.